xml/depl_cisco_opflex.xml

```diff
@@ -0,0 +1,686 @@
+   <figure>
+     <title>Create Allow policy.</title>
+     <mediaobject>
+      <imageobject role="fo">
+       <imagedata fileref="gbp2.png" width="100%" format="png"/>
+      </imageobject>
+      <imageobject role="html">
+       <imagedata fileref="gbp2.png" width="75%" format="png"/>
+      </imageobject>
+     </mediaobject>
+ </figure>
+ 
+ <para>
+     In <literal>Policy | Application Policy | Policy Classifiers</literal>, create two bi-directional classifiers, one for TCP and the other for the ICMP protocol.
+ </para>
+ 
+    <figure>
+     <title>Creating a TCP classifier.</title>
+     <mediaobject>
+      <imageobject role="fo">
+       <imagedata fileref="gbp3.png" width="100%" format="png"/>
+      </imageobject>
+      <imageobject role="html">
+       <imagedata fileref="gbp3.png" width="75%" format="png"/>
+      </imageobject>
+     </mediaobject>
+ </figure>
+ 
+ <para>
+     In <literal>Policy | Application Policy | Policy Rules</literal> create two rules, one for TCP and one for ICMP, linking the Policy classifier to the Allow policy action.
+ </para>
+ 
+     <figure>
+     <title>Linking the Policy classifier to the Allow policy.</title>
+     <mediaobject>
+      <imageobject role="fo">
+       <imagedata fileref="gbp4.png" width="100%" format="png"/>
+      </imageobject>
+      <imageobject role="html">
+       <imagedata fileref="gbp4.png" width="75%" format="png"/>
+      </imageobject>
+     </mediaobject>
+ </figure>
+ 
+ <para>
+     In <literal> Policy | Application Policy | Policy Rule Set</literal> create a Dbaccess rule set that links to the TCP and ICMP rules.
+ </para>
+ 
+     <figure>
+     <title>Link Dbaccess to TCP and ICMP rules.</title>
+     <mediaobject>
+      <imageobject role="fo">
+       <imagedata fileref="gbp5.png" width="100%" format="png"/>
+      </imageobject>
+      <imageobject role="html">
+       <imagedata fileref="gbp5.png" width="75%" format="png"/>
+      </imageobject>
+     </mediaobject>
+ </figure>
+ 
+ <para>
+     In <literal>Policy | Groups</literal> create two internal groups, Web and DB.
+ </para>
+ 
+ <procedure>
+   <step>
+    <para>
+     Edit each group so that the DB group is the rule set provider, and the Web group is the rule set consumer.
+    </para>
+   </step> 
+   
+   <step>
+    <para>
+        Click on the Web group and the DB group, and then select <literal>Create Member</literal> to deploy two instances for each group.
+    </para>
+   </step>
+   
+      <step>
+    <para>
+       Confirm that a Web instance can ping its gateway, the other Web instance and the DB instances.
+    </para>
+   </step>
+   
+  <step>
+    <para>
+       For further testing, remove the Dbaccess rule set from the Consumed Rule Set for the Wweb group to see the pings stop. Re-add the Dbaccess rule set to see the pings start again.
+    </para>
+   </step>
+   </procedure>
+ </sect1>
+ 
+ <sect1 xml:id="sec.troubleshooting">
+     <title>Troubleshooting</title>
+     <para>
```

Comment:
I suggest to turn this into a Q&A section. This allows to add more topics in the future and also makes it possible to link to each question. See xml/depl_troubleshooting.xml for a blueprint.